controllers/storagecluster/cephcluster.go

```diff
@@ -1184,6 +1200,44 @@ func (r *StorageClusterReconciler) setReadAffinity(sc *ocsv1.StorageCluster) err
 	return nil
 }
 
+func (r *StorageClusterReconciler) setCephFSKernelMountOption(sc *ocsv1.StorageCluster) error {
+	var cephFSKernelMountOption = "ms_mode=prefer-crc"
+	var restartNeeded = false
+	// If In-transit encryption is enabled set cephFsKernelMountOption to "ms_mode=secure"
+	if sc.Spec.Network != nil && sc.Spec.Network.Connections != nil && sc.Spec.Network.Connections.Encryption != nil && sc.Spec.Network.Connections.Encryption.Enabled {
+		cephFSKernelMountOption = "ms_mode=secure"
+	}
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      util.OcsOperatorConfigName,
```

Comment:
This configmap appears to be created/updated in multiple places, here and in ocsoperator_config.go. Will that not cause conflicts?

Reply:
Going forward ocsinitializer will just create the configmap with the keys so that rook operator doesn't through errors. (as rook operator expects these value so if the configmap or the key doesn't exist it will stay in container create config error). So we need to create the configmap with some default value when the operator is installed & we can later update them with correct values in storagecluster controller.